pages/portfolio/index.vue

```diff
@@ -50,274 +60,182 @@
+              class="w-90vw m-auto flex md:block lg:block mt-20"
+            />
+          </transition>
         </div>
       </div>
-    </transition>
-    <div class="text-center mt-20">
-      <h2
-        v-if="showCard"
-        class="text-center text-3xl md:text-4xl lg:text-6xl mt-50"
-      >
-        Software Development Group
-      </h2>
-      <h2
-        v-if="showCpCard"
-        class="text-center text-3xl md:text-4xl lg:text-6xl mt-50"
-      >
-        Competitive Programming Group
-      </h2>
-      <h2
-        v-if="showMlCard"
-        class="text-center text-3xl md:text-4xl lg:text-6xl mt-50"
-      >
-        Machine Learning Group
-      </h2>
-      <h2
-        v-if="showInfosecCard"
-        class="text-center text-2xl md:text-4xl lg:text-6xl mt-50"
-      >
-        InfoSec Group
-      </h2>
-      <div v-for="member in members" id="mem" :key="member" class="m-auto">
-        <transition appear @before-enter="beforeEnter" @enter="enter">
-          <sdgCard
-            v-show="showCard"
-            class="w-120 md:w-220 lg:w-320 m-auto px-10 flex md:block lg:block"
-          />
-        </transition>
-        <transition appear @before-enter="beforeEnter" @enter="enter">
-          <cpCard
-            v-show="showCpCard"
-            class="w-120 md:w-220 lg:w-320 m-auto px-10 flex md:block lg:block"
-          />
-        </transition>
-        <transition appear @before-enter="beforeEnter" @enter="enter">
-          <mlCard
-            v-show="showMlCard"
-            class="w-120 md:w-220 lg:w-320 m-auto px-10 flex md:block lg:block"
-          />
-        </transition>
-        <transition appear @before-enter="beforeEnter" @enter="enter">
-          <infosecCard
-            v-show="showInfosecCard"
-            class="w-120 md:w-220 lg:w-320 m-auto px-10 flex md:block lg:block"
-          />
-        </transition>
-      </div>
-    </div>
+    </main>
   </section>
 </template>
 <script>
 import { defineComponent } from '@nuxtjs/composition-api';
+import lottie from 'vue-lottie/src/lottie.vue';
+import * as devAnimationData from '~/assets/55885-developer.json';
+import * as alumniAnimationData from '~/assets/30304-back-to-school.json';
 export default defineComponent({
-  components: {},
+  components: {
+    lottie,
+  },
   data() {
     return {
       members: 8,
+      alums: 8,
       showCard: false,
-      showCpCard: false,
-      showMlCard: false,
-      showInfosecCard: false,
+      showAlum: false,
+      anim: {
+        firstAnim: null,
+        secondAnim: null,
+      },
+      lottieOptions: {
+        first: { animationData: devAnimationData.default },
+        second: { animationData: alumniAnimationData.default },
+      },
     };
   },
   mounted() {
     this.$nextTick(() => {
       this.$nuxt.$loading.start();
       setTimeout(() => this.$nuxt.$loading.finish(), 1000);
     });
+    this.fetchMembers();
+    this.fetchAlums();
```

Comment:
(Fetch)[https://nuxtjs.org/docs/features/data-fetching] with `keep-alive` will allow us a better caching mechanism otherwise each mounted call will read the json which isn't performant.